model/task/task.go

```diff
@@ -741,6 +741,7 @@ func (t *Task) MarkEnd(finishTime time.Time, detail *apimodels.TaskEndDetail) er
 
 	t.TimeTaken = finishTime.Sub(t.StartTime)
 	t.Details = *detail
+	t.Aborted = false
```

Comment:
I think it might make sense to move this to https://github.com/evergreen-ci/evergreen/pull/2395/files#diff-26202a3e043071277c38c291fbca1260R420, or add a comment there, since it won't be obvious to later readers that calling `t.MarkEnd` from `MarkEnd` clears the aborted flag, and therefore that `event.LogTaskFinished(t.Id, t.Execution, t.HostId, status, t.Aborted)` must be called before `t.MarkEnd`. This seems like a potential source of future bugs, since changing the order will change the abort boolean passed to `LogTaskFinished`.

Reply:
I pulled setting the aborted field out of t.MarkEnd